test/Constraints/members.swift

```diff
@@ -590,3 +590,10 @@ do {
   throw SR_2193_Error.Boom
 } catch let e as SR_2193_Error.Boom { // expected-error {{enum element 'Boom' is not a member type of 'SR_2193_Error'}}
 }
+
+// rdar://problem/25341015
+extension Sequence {
+  func r25341015() -> Int {
+    return max(1, 2) // expected-error {{member 'Sequence.max' shadows global function 'Swift.max'}}
```

Comment:
With a custom protocol you could also demonstrate the `decl_declared_here` note. It might be best for the test to include an example with more than just 2 candidates.